beet/resources/pack_format_registry.json

```diff
@@ -0,0 +1,5112 @@
+    "release_target": "1.14.2",
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 1962,
+    "protocol_version": 484,
+    "data_pack_version": 4,
+    "resource_pack_version": 4,
+    "build_time": "2019-05-27T07:19:35+00:00",
+    "release_time": "2019-05-27T07:21:11+00:00",
+    "sha1": "75bf72d8e4b38cf5f729637efc116697edec4334"
+  },
+  {
+    "id": "1.14.2-pre3",
+    "name": "1.14.2 Pre-Release 3",
+    "release_target": "1.14.2",
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 1960,
+    "protocol_version": 483,
+    "data_pack_version": 4,
+    "resource_pack_version": 4,
+    "build_time": "2019-05-22T13:11:15+00:00",
+    "release_time": "2019-05-22T13:12:51+00:00",
+    "sha1": "4a0126da466f90d6e52eb3e27332f7f7070f4be8"
+  },
+  {
+    "id": "1.14.2-pre2",
+    "name": "1.14.2 Pre-Release 2",
+    "release_target": "1.14.2",
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 1959,
+    "protocol_version": 482,
+    "data_pack_version": 4,
+    "resource_pack_version": 4,
+    "build_time": "2019-05-17T12:19:26+00:00",
+    "release_time": "2019-05-17T12:21:03+00:00",
+    "sha1": "dee854f57cac0fc9f1b96b96254a67829dd103ae"
+  },
+  {
+    "id": "1.14.2-pre1",
+    "name": "1.14.2 Pre-Release 1",
+    "release_target": "1.14.2",
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 1958,
+    "protocol_version": 481,
+    "data_pack_version": 4,
+    "resource_pack_version": 4,
+    "build_time": "2019-05-16T15:38:50+00:00",
+    "release_time": "2019-05-16T15:40:25+00:00",
+    "sha1": "0525ae4750d18d6c3cf9edee1f510333919c6003"
+  },
+  {
+    "id": "1.14.1",
+    "name": "1.14.1",
+    "release_target": "1.14.1",
+    "type": "release",
+    "stable": false,
+    "data_version": 1957,
+    "protocol_version": 480,
+    "data_pack_version": 4,
+    "resource_pack_version": 4,
+    "build_time": "2019-05-13T11:08:37+00:00",
+    "release_time": "2019-05-13T11:10:12+00:00",
+    "sha1": "183e49ba0fabb5ef8a304f8f5907fa887f40c466"
+  },
+  {
+    "id": "1.14.1-pre2",
+    "name": "1.14.1 Pre-Release 2",
+    "release_target": "1.14.1",
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 1956,
+    "protocol_version": 479,
+    "data_pack_version": 4,
+    "resource_pack_version": 4,
+    "build_time": "2019-05-09T13:59:28+00:00",
+    "release_time": "2019-05-09T14:01:04+00:00",
+    "sha1": "1d0ccfff5f8d13d3ecaddbc481a9a4815e312e4e"
+  },
+  {
+    "id": "1.14.1-pre1",
+    "name": "1.14.1 Pre-Release 1",
+    "release_target": "1.14.1",
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 1955,
+    "protocol_version": 478,
+    "data_pack_version": 4,
+    "resource_pack_version": 4,
+    "build_time": "2019-05-07T14:43:03+00:00",
+    "release_time": "2019-05-07T14:44:42+00:00",
+    "sha1": "656e98e9c5a418b4747b0e5892fd301963b0bcc7"
+  },
+  {
+    "id": "1.14",
+    "name": "1.14",
+    "release_target": "1.14",
```

Comment:
Beet technically supported down to 1.13 before. I'm not sure if we care about this regression?